content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -122,7 +122,6 @@ different Kubernetes components.
 | `QOSReserved` | `false` | Alpha | 1.11 | |
 | `RemainingItemCount` | `false` | Alpha | 1.15 | |
 | `ResourceLimitsPriorityFunction` | `false` | Alpha | 1.9 | |
-| `RotateKubeletClientCertificate` | `true` | Beta | 1.8 | |
 | `RotateKubeletServerCertificate` | `false` | Alpha | 1.7 | 1.11 |
```

Comment:
You also need to move this line, right?

Reply:
Ah no. But we _should_ have a line for when *RotateKubeletClientCertificate* was alpha. Unless it went insta-beta.

Reply:
1.8 was a long history to dig back. We have no idea whether there was an Alpha at all. Even if there WAS one, what do we gain by adding a line for pre-1.8 release? We are not longer supporting it, right?

Reply:
It's a minor thing for sure.